Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -45,6 +45,8 @@ describe(
     const boxBackFaceCullingUrl =
       "./Data/Models/Box-Back-Face-Culling/Box-Back-Face-Culling.gltf";
     const boxBackFaceCullingOffset = new HeadingPitchRange(Math.PI / 2, 0, 2.0);
+    const morphPrimitivesTestUrl =
+      "./Data/Models/GltfLoader/MorphPrimitivesTest/glTF/MorphPrimitivesTest.gltf";
```

Comment:
I see we only really use the glTF version of these assets. I would suggest two things:

1. Add a test for the one quantized model, that _should_ work since `KHR_mesh_quantization` is applied well before we get to the morph targets stage, but a unit test will let us prove it
2. The other models (like the binary ones) that are unused can be removed from this PR (please squash your commits to keep unused binary files from the history)

Reply:
For now the quantized model won't work because it's animated, but I'll leave it in the directory for future testing.